### PR TITLE
feat: working on the svgr package

### PR DESCRIPTION
**Pull request by im-klau**
## Jira issue

JIRA issue: [FSU-58](https://klau.atlassian.net/browse/FSU-58)

## Describe your changes
This PR adds the `svgr` package and configures it with Storybook. The package transforms a regular svg code into React component. Take a look at the playground to understand what I mean - https://react-svgr.com/playground/

Additionally, there's a new `Icon` component that handles the 3 basic props: `name`, `color`, and `size`.
It renders a proper icon based on the `name` prop. The icon must be imported and included in the `icons` object first.

Feel free to read the README file to learn how to add a new icon to the repo.

Also, check out the `Icons` Storybook page to see our icon gallery (there's only a heart icon 😅).

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Review by nconnect1789 — APPROVED** [review]
Great job 🚀

File: packages/design-system/components/icon/Icon.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const Icon = ({ name, color, size }) => {
+  const IconComponent = icons[name]
+
+  return (
+    <IconComponent color={color} width={sizes[size]} height={sizes[size]} />
+  )
+}
+
+Icon.propTypes = {
+  name: oneOf(["heart"]).isRequired,
```
Comment (paulgrym) [inline packages/design-system/components/icon/Icon.jsx:24]: Can we use Object.keys as you did in breakpoints PR? Something like:
```
const getIconsNames = (projectIcons) => {
  let iconsNames = []

  Object.keys(projectIcons).forEach((key) => {
    iconsNames = [...iconsNames, key.toString()]
  })

  return iconsNames
}
```

and then:
```
Icon.propTypes = {
  name: oneOf(getIconsNames(icons)).isRequired,
...
```
Would it make sense?

File: packages/design-system/icons/__stories__/Icons.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Meta, IconGallery, IconItem } from "@storybook/addon-docs"
+
+import HeartIcon from "../Heart.svg"
+
+<Meta title="Icons" />
+
+# Icons
+
+A list of all available icons in the project.
```
Comment (paulgrym) [inline packages/design-system/icons/__stories__/Icons.stories.mdx:9]: I just love that we will be able to see them all in one place!

**Review by paulgrym — APPROVED** [review]
Great job - as always!

File: packages/design-system/README.md [hunk 3]
```diff
@@ -1 +1,26 @@
 # Design System for web pages
+
+### Adding a new SVG icon
+
+1. Export SVG code from the design file
+2. Use the [SVGO tool](https://jakearchibald.github.io/svgomg/) to optimize the svg code
+3. Create a new `.svg` file in the `design-system/icon` directory with a suitable icon name
```
Comment (im-klau) [inline packages/design-system/README.md:7]:
```suggestion
3. Create a new `.svg` file in the `design-system/icons` directory with a suitable icon name
```

File: packages/design-system/components/icon/Icon.jsx (hunk 1 above)
Comment (im-klau) [reply packages/design-system/components/icon/Icon.jsx:24]: Yessss, definitely makes sense! 🙌 I'll just use a simpler solution, but thank you!

**Pull request merged**